src/components/ProfileImg/ProfileImg.jsx

```diff
@@ -8,6 +8,7 @@ function ProfileImg({ avatar, userId }) {
   const [profileImg, setProfileImg] = useState('');
 
   useEffect(() => {
+    if(userId) {
```

Comment:
Won't ask you to change this as we're done with the project now, but just to be aware, you can do this is a little bit of a cleaner way: 

```js
if(!userId) {
   return
}
```

This is what we call a guard statement, where it will stop any code from being executed underneath it if the userId is falsy